code/Silvergraph.php

```diff
@@ -204,7 +200,7 @@ public function dot(){
 
         // Defend against source_file_comments
         Config::nest();
-        Config::inst()->update('SSViewer', 'source_file_comments', false);
+        Config::inst()->update('SilverStripe\\View\\SSViewer', 'source_file_comments', false);
```

Comment:
As above

Reply:
This is not exactly the same (there is no `use SilverStripe\View\SSViewer` before) but I think that making that dependency explicit is a good move anyway. :+1: